src/components/wagonwheelComponent/index.js

```diff
@@ -39,6 +48,8 @@ class WagonWheelComponent extends Component {
     selectTeam (e) {
         const team_id =  parseInt(e.target.selectedOptions[0].value);
```

Comment:
Follow consistent naming convention. For variables always use camelCase with first letter in lower case.